src/ads/components/ads-label.js

```diff
@@ -1,7 +1,8 @@
-vjs.AdsLabel = vjs.Component.extend({
+var vjsComponent = videojs.getComponent('Component');
+var AdsLabel = videojs.extend(vjsComponent, {
   /** @constructor */
-  init: function (player, options) {
-    vjs.Component.call(this, player, options);
+  constructor: function (player, options) {
+    vjsComponent.call(this, player, options);
 
```

Comment:
Ok then I suggest that you merge my other pull request https://github.com/MailOnline/videojs-vast-vpaid/pull/105 in the master branch that contains fixes that make the debug page usable, then I hope those changes will disappear from this pull request (hehe) as they will already be merged, then you can make a branch for 5.x and merge this pull request inside.

After that, I'll probably add more features in the master branch, that we'll merge into 5.x branch when needed.

Sounds good ?